bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -68,13 +68,10 @@ <h2 class="mzp-c-card-title"><span>{{ title }}</span></h2>
     {% endif %}
     </div>
   </a>
-  {% if video_url %}
+  {% if youtube_id %}
   <div class="mzp-c-card-video-wrapper hidden">
     <figure class="mzp-c-card-video-content">
-      <video class="mzp-c-card-video" preload="none" controls playsinline data-ga-label="{{ ga_title }}">
-        <source src="{{ video_url }}.webm" type="video/webm">
-        <source src="{{ video_url }}.mp4" type="video/mp4">
-      </video>
+      <div class="video-play" data-id="{{ youtube_id }}"><h2>test</h2></div>
```

Comment:
Nit: remove `<h2>test</h2>`?

Reply:
@alexgibson thanks for the notes, Alex! 😓 Truth be told, I pushed what I had EOD so it is a bit rough in places.